src/clj/clojurians_log/routes.clj

```diff
@@ -42,13 +42,15 @@
 
       (let [db       (d/db conn)
             messages (queries/channel-day-messages db channel date)
+            thread-messages (queries/channel-thread-messages-of-day db channel date)
             user-ids (slack-messages/extract-user-ids messages)]
         (-> request
             context
             (assoc :data/channel (queries/channel db channel)
                    :data/channels (queries/channel-list db date)
                    :data/messages messages
-                   :data/target-message (some #(when (= (:message/ts %) ts) %) messages)
+                   :data/thread-messages (group-by :message/thread-ts thread-messages)
```

Comment:
I would do the data munging here, and add these onto the individual messages, rather than placing that extra burden on the views.

So only have `:data/messages` (seq of non-child messages), and have give each message a `:message/children` key which is a seq of its child messages.